tensorflow/python/eager/context.py

```diff
@@ -1430,10 +1437,16 @@ def _initialize_physical_devices(self, reinitialize=False):
       self._physical_device_to_index = {
```

Comment:
Will you need this index map too? If so, maybe we should add pluggable devices directly to `_physical_devices`.

Reply:
_physical_devices already contains the pluggable devices, the pluggable device list here is to justify whether the device got from _physical_devices is a PluggableDevice.
for example:
```
1639     if dev.device_type != "GPU" and dev not in self._pluggable_devices:
1640       raise ValueError("Cannot set memory growth on non-GPU and non-Pluggable devices")

```